src/main/java/springboot/model/Book.java

```diff
@@ -7,11 +7,9 @@
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
-import lombok.Getter;
-import lombok.Setter;
+import lombok.Data;
 
-@Getter
-@Setter
+@Data
```

Comment:
```suggestion
@Getter
@Setter
```

Reply:
why ?

Reply:
https://medium.com/@miguelangelperezdiaz444/the-hidden-dangers-of-lombok-annotations-in-your-java-code-what-you-need-to-know-8acdce2d6b89